docs/docs-content/troubleshooting/generate-har-files.md

```diff
@@ -0,0 +1,155 @@
+:::
+
+This guide explains how you can generate HAR files in Chrome, Safari, and Firefox and sanitize the sensitive data they contain.
+
+## Generate HAR Files
+
+<Tabs>
+  <TabItem value="chrome" label="Chrome" default>
+    1. Open the page with issues in Chrome.
+    2. On your keyboard, open Developer Tools:
+       - For Windows and Linux, press **F12** or **Ctrl + Shift + I**.
+       - For Mac, press **Fn + F12** or **Cmd + Option + I**.
+
+       For alternative methods to open Developer Tools, refer to [Open Chrome DevTools](https://developer.chrome.com/docs/devtools/open).
+    3. In Developer Tools, select the **Network** tab. Then, on the **Network** pane:
+       - Toggle <RecordNetworkLog /> to <FontAwesomeIcon icon={faCircleStop} color={"#E46962"} /> to start recording network activity.
+       - Click <FontAwesomeIcon icon={faBan} rotation={270} /> to clear the current network log.
+       - Select the **Preserve log** and **Disable cache** checkboxes.
+
+       ![](/assets/docs/images/troubleshooting/generate-har-files/chrome-dev-tools-settings.png)
+    4. On the page, reproduce the issues you've encountered.
+    5. In the network log, right-click any item and select **Save all as HAR with content**.
+  </TabItem>
+  <TabItem value="safari" label="Safari">
+    1. If you haven't yet, [enable the web development features in Safari](https://developer.apple.com/documentation/safari-developer-tools/enabling-developer-features).
+    2. Open the page with issues.
+    3. On your keyboard, press **⌥ + ⌘ + I** to open Web Inspector.
+    4. Select the **Network** tab, and, in the **Network** tab menu:
+       - Press **⌘ + K** to clear the current network items.
+       - Next to the **All** drop-down menu, click the **Other filter options** icon > **Preserve Log**. 
+       - Select the **Disable Caches** checkbox.
+
+       ![](/assets/docs/images/troubleshooting/generate-har-files/safari-web-inspector.png)
+    5. On the page, reproduce the issues you've encountered.
+    6. In the **Network** tab menu, select **Export**.
+  </TabItem>
+  <TabItem value="firefox" label="Firefox">
+    1. Open the page with issues in Firefox.
+    2. On your keyboard, open Developer Tools:
+       - For Windows and Linux, press **F12** or **Ctrl + Shift + I**.
+       - For Mac, press **Fn + F12** or **Cmd + Option + I**.
+
+       For alternative methods to open Developer Tools, refer to [Firefox DevTools User Docs](https://firefox-source-docs.mozilla.org/devtools-user/).
+    3. In Developer Tools, select the **Network** tab and, in the **Network** tab menu:
+       - Select <FontAwesomeIcon icon={faTrashCan} /> to clear the current network log.
+       - Toggle <FontAwesomeIcon icon={faPlay} color={"#007AFF"} /> to <FontAwesomeIcon icon={faPause} /> to start recording network activity.
```

Comment:
`▶️ to ⏸️ to start recording network activity.` 

Confusing flow which tries to explain how to toggle recording network activity.

Could be along the lines of

`Utilize ⏸️ or ▶️ to toggle the recording of network activity.`
or
`To Toggle the recording of the network activity utilize ▶️ to start and ⏸️ to pause`.